getting-started.adoc

```diff
@@ -1,154 +1,192 @@
 [[getting-started-with-raincatcher]]
```

Comment:
the old toc was very markdown specific, currently we don't have a replacement, but I hope we can use asciibinder to give us a useful toc, eg https://docs.openshift.com/container-platform/3.4/architecture/infrastructure_components/kubernetes_infrastructure.html